controllers/machinehealthcheck_controller.go

```diff
@@ -220,6 +220,13 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	m.Status.CurrentHealthy = int32(len(healthy))
 
 	var unhealthyLimitKey, unhealthyLimitValue interface{}
+	if m.Spec.UnhealthyRange == nil {
+		unhealthyLimitKey = "max unhealthy"
+		unhealthyLimitValue = m.Spec.MaxUnhealthy
+	} else {
+		unhealthyLimitKey = "unhealthy range"
+		unhealthyLimitValue = *m.Spec.UnhealthyRange
+	}
```

Comment:
Do we have to optimize here? Could we just call the logger twice in two `ifs` below? We could potentially use a switch case if readability is a concern.

Reply:
I'm not sure I follow, I updated to call the logger twice. PTAL.